### PR TITLE
Allow user to force using the current develop branch version when merging a feature

### PR DESCRIPTION
**Pull request by bcluap**
This is a fix for https://github.com/aleksandr-m/gitflow-maven-plugin/issues/238

Also note this is not just new functionality. This also fixes a bug where the feature version was not being correctly handled if skipTestProject=true

### CODE REVIEW

**Comment by aleksandr-m** [comment]
@bcluap Please commit your specific changes in different branch. You can force push to this one or create another PR.

**Comment by aleksandr-m** [comment]
@bcluap Can you clean this up?

**Comment by aleksandr-m** [comment]
@bcluap Are you still interested in this feature? If yes, can you clean this PR so I can review it.

File: src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowFeatureFinishMojo.java [hunk 1]
```diff
@@ -89,6 +89,13 @@ public class GitFlowFeatureFinishMojo extends AbstractGitFlowMojo {
     @Parameter(property = "postFeatureFinishGoals")
     private String postFeatureFinishGoals;
 
+    /**
+     * Whether to always use the develop branch version when merging into it.
+     *
+     * @since 1.14.1
+     */
+    @Parameter(property = "retainDevelopVersion", defaultValue = "true")
```
Comment (AdrienHorgnies) [inline src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowFeatureFinishMojo.java:97]: @aleksandr-m Why would we ever use a version different than develop's version ?

**Comment by aleksandr-m** [comment]
Pretty old one, let's focus on #367.

**Pull request closed without merging**